### PR TITLE
Lic 339 search simplified

### PR DESCRIPTION
**Pull request by atodd-noms**

### CODE REVIEW

File: server/views/search/results.pug [hunk 1]
```diff
@@ -0,0 +1,18 @@
+extends ../layout
+
+block content
+
+    div.pure-g.pure-u-1
```
Comment (romidane) [inline server/views/search/results.pug]:
```
div.pure-g  // (grid container)
  div.pure-u-1  // (grid unit)
```

File: test/utils/searchParserTest.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+        inputsAndOutputs
+            .forEach(([label, searchTerms, expected]) => {
+                it(`should report error when ${label}`, () => {
+                    expect(parseSearchTerms(searchTerms).error).to.eql(expected);
+                });
+            });
+    });
+
+    // describe('parsing and url encoding names and special characters NOT including diacritics', () => {
```
Comment (romidane) [inline test/utils/searchParserTest.js]: Do we need this code?

File: server/routes/search.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    router.use(function(req, res, next) {
+        if (typeof req.csrfToken === 'function') {
+            res.locals.csrfToken = req.csrfToken();
+        }
+        next();
+    });
+
+    router.get('/offender', asyncMiddleware(async (req, res) => {
+        logger.debug('GET /search/offender');
```
Comment (whitfield-mj) [inline server/routes/search.js]: Doesn't need to be async or use the middleware

File: server/routes/search.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        res.render('search/offender');
+    }));
+
+    router.post(['/offender', '/offender/results'], asyncMiddleware(async (req, res) => {
+        logger.debug('POST /search/offender');
```
Comment (whitfield-mj) [inline server/routes/search.js]: as above

**Pull request merged**